static/app/views/performance/transactionSummary/transactionOverview/content.tsx

```diff
@@ -344,6 +378,83 @@ function SummaryContent({
     organization.features.includes('performance-spans-new-ui') &&
     organization.features.includes('insights-initial-modules');
 
+  // This is adapted from the `getEventFieldValues` function in `events/searchBar.tsx`
+  const getTransactionFilterTagValues = memoize(
+    (tag, queryString): Promise<string[]> => {
+      const projectIdStrings = eventView.project?.map(String);
+
+      if (isAggregateField(tag.key) || isMeasurement(tag.key)) {
+        // We can't really auto suggest values for aggregate fields
+        // or measurements, so we simply don't
+        return Promise.resolve([]);
+      }
+
+      // device.class is stored as "numbers" in snuba, but we want to suggest high, medium,
+      // and low search filter values because discover maps device.class to these values.
+      if (isDeviceClass(tag.key)) {
+        return Promise.resolve(DEVICE_CLASS_TAG_VALUES);
+      }
+
+      const fetchPromise = fetchTagValues({
+        api,
+        orgSlug: organization.slug,
+        tagKey: tag.key,
+        search: queryString,
+        projectIds: projectIdStrings,
+        // allows searching for tags on transactions as well
+        includeTransactions: true,
+        // allows searching for tags on sessions as well
+        includeSessions: true,
+      });
+
+      return fetchPromise.then(
+        results => results.filter(({name}) => defined(name)).map(({name}) => name),
+        () => {
+          throw new Error('Unable to fetch event field values');
+        }
+      );
+    },
+    ({key}, queryString) => `${key}-${queryString}`
+  );
+
+  function renderSearchBar() {
+    return (
+      <StyledSearchBarWrapper>
+        <Feature
```

Comment:
nit: in most cases we aren't using this component anymore, it's simple enough to just do `organization.features.includes('search-query-builder-performance')`